crates/bin/prove_block/src/rpc_utils.rs

```diff
@@ -12,24 +12,36 @@ use starknet_types_core::felt::Felt;
 
 /// A `Storage` impl backed by RPC
 #[derive(Clone)]
-pub(crate) struct RpcStorage {}
+pub(crate) struct RpcStorage {
+    cache: HashMap<Vec<u8>, Vec<u8>>,
```

Comment:
TODO: this is redundant with the `CompositeStorage` setup, needs to be removed

Reply:
Yes, I don't see the point of this?